apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/exception/Exceptions.kt

```diff
@@ -98,11 +98,24 @@ class CacheMissException(val key: String, val fieldName: String? = null) : Apoll
 class HttpCacheMissException(message: String, cause: Exception? = null) : ApolloException(message = message, cause = cause)
 
 /**
- * Multiple exceptions happened, for an exemple with a [CacheFirst] fetch policy
+ * Multiple exceptions happened, for an example with a [CacheFirst] fetch policy
  */
-class ApolloCompositeException(first: Throwable?, second: Throwable?) : ApolloException(message = "multiple exceptions happened", second) {
-  val first = (first as? ApolloException) ?: throw RuntimeException("unexpected first exception", first)
-  val second = (second as? ApolloException) ?: throw RuntimeException("unexpected second exception", second)
+class ApolloCompositeException(
+    private val original: Throwable?, private val suppressed: Throwable?
```

Comment:
This is a Kotlin source incompatible change because of named parameters. Can you keep `first` and `second`?

Reply:
okay

Reply:
accessing first and second might throw RuntimeException so it would easy if client is aware when using such properties.
But throws annotation is not applicable to target 'member property without backing field or delegate' that is why 
1. property getter is added
2. getter function cannot access non val constructor params
3. params marked val must name different then properties name

That's the reason had to rename constructor params and set them to private val

Reply:
I see 👍 . We're not declaring the `@Throws` exceptions consistently in the codebase. Mainly because Kotlin doesn't have checked exceptions but that'll help with Java interop if someone ever watch to catch these. So works for me 👍

Reply:
Removed Throws annotation as it only available in jvm - https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-throws/